src/js/components/GlobalSearch.jsx

```diff
@@ -35,6 +35,7 @@ class GlobalSearch extends Component {
           cache={false}
           options={[]}
           showValueTooltip
+          noResultsText={`${this.state.inputValue.length >= 3 ? 'No results found' : ''}`}
```

Comment:
Careful with the 3 here. Since this value is now a runtime application property (openboxes.typeahead.minLength Config.groovy), we probably need a way for the backend to communicate the global application properties to the UI (React) so that we don't get out of sync. I could imagine some type of app context (API endpoint) that gets pulled in by the React app on initialization.